website/docs/developer/research/research-graphql.md

```diff
@@ -118,18 +118,105 @@ Details on creating and configuring a GraphQL API using AppSync can be found [he
 - [Graphehe](https://github.com/graphql-python/graphene)- Server using Python. Most popular and contributors.
 - [Ariadne](https://ariadnegraphql.org/)- Server using Python
 - [Strawberry](https://strawberry.rocks/)- Server using Python
+- [CMR-GraphQL](https://git.earthdata.nasa.gov/projects/CMRQL/repos/hackfest-cmr-graphql/browse)- Currently being used by NASA GHRC developers Check out the example of their GraphQL schema and resolver [here](https://git.earthdata.nasa.gov/projects/CMRQL/repos/hackfest-cmr-graphql/browse/cmr-graphql/graphql/graphql.js)
 
 
 ### GraphQL in ORCA
 
-Some of the lambdas that will be affected are:
+Some of the lambdas that might be affected are:
 - post_copy_request_to_queue
 - db_deploy
 - post_to_database
 - request_files
 - copy_files_to_archive
 - request_status_for_granule
-- request_status_for_granule
+- request_status_for_job
+- db_deploy
+- post_to_database
+
+:::important
+Apart from updating those lambdas, developers need to create the GraphQL endpoint using terraform or AWS SAM, update requirements.txt, requirements-dev.txt or bin/build.sh, bin/run_tests.sh by adding additional dependiencies like `graphene` in case of using graphene, or `npm install apollo-server graphql` in case of using Apollo server.
+In case of using Javascript libraries like Apollo server, additional files/codes are needed to write the [schema](https://github.com/serverless/serverless-graphql/blob/master/app-backend/dynamodb/schema.js) and [resolver](https://github.com/serverless/serverless-graphql/blob/master/app-backend/dynamodb/resolvers.js).
+In case of using Python library like [Graphene](https://github.com/graphql-python/graphene), developers need to update .py files to import libraries, create class and queries, and to create schema and resolver. 
+:::
+  
+  
+  A few suggestions are given below:
+```commandline
```

Comment:
I would reformat this section to follow the styling guidelines.
Change your separations like these
`#---------------------------post_copy_request_to_queue----------------------`
to be a heading 4
`#### post_copy_request_to_queue`
Change your commented out items like `## Additional changes are expected.` to be bullets like `- Additional changes are expected.`


Try to reserve the code blocks for code or command line commands and not notes like below if possible unless providing actual commands. This would be for lines 145-186